scripts/repo-utils/index.cjs

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-undef */
+
+const path = require("path");
+const { readFileSync } = require("fs");
+const { fileURLToPath } = require("url");
+const { createRequire } = require("module");
+const semver = require("semver");
+
+let USE_ESM = false;
+try {
+  const type = readFileSync(
+    path.join(__dirname, "../../.module-type"),
+    "utf-8"
+  ).trim();
+  USE_ESM = type === "module";
+} catch (_) {}
+
+exports.USE_ESM = USE_ESM;
+
+if (typeof jest !== "undefined") {
+  exports.itNoESM = USE_ESM ? it.skip : it;
+  exports.itESM = USE_ESM ? it : it.skip;
+  exports.itGteESM = function (version) {
+    return USE_ESM && semver.gte(process.version, version) ? it : it.skip;
+  };
+  exports.itGteNoESM = function (version) {
+    return !USE_ESM && semver.gte(process.version, version) ? it : it.skip;
+  };
+  exports.itGte = function (version) {
+    return semver.gte(process.version, version) ? it : it.skip;
+  };
+}
+
+exports.globals = function (metaUrl) {
```

Comment:
I'd prefer to call this `commonJS`, since they are not globals :)

Reply:
Sounds good! I'm a mess when it comes to naming things. :)